apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerConnection.cs

```diff
@@ -116,6 +119,27 @@ public override ConnectionState State
         /// </summary>
         public QueryOptions QueryOptions { get; set; }
 
+        /// <summary>
+        /// Request CommitStats for all read/write transactions throughout the
+        /// lifetime of the connection.
+        /// </summary>
+        public bool ReturnCommitStats { get; set; }
+
+        /// <summary>
+        /// The response of the last commit on this connection. This includes the
+        /// commit response of any implicit commits by commands that are executed
+        /// without a transaction.
+        ///
+        /// The property is null after the execution of a command that uses a
+        /// transaction, and will return the commit response of that transaction
+        /// when the transaction is committed.
+        /// </summary>
+        public CommitResponse LastCommitResponse
```

Comment:
This can just be an automatically implemented property, so we don't need the field declaration.